IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -46,6 +46,13 @@
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->